pyscriptjs/src/pyscript.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "pyscript",
+    "description": "default config",
+    "version": "0.1",
+    "type": "app",
+    "author_name": "madhur",
+    "author_email": "madhurtandon23@gmail.com",
```

Comment:
`hello@pyscript.com`

Reply:
we need to make sure that email exists though, does it?

Reply:
Exactly... but it mustn't be a personal email from our team. 🤣 

Alternatively we could use `foo@bar.com` (see: https://www.bar.com/foo)

Reply:
using `foo@bar.com`